resources/services/wafv2/web_acls.go

```diff
@@ -347,14 +343,38 @@ func resolveWafv2webACLResourcesForWebACL(ctx context.Context, meta schema.Clien
 	client := meta.(*client.Client)
 	service := client.Services().WafV2
 
-	// Resolve resources that are associated with the given web ACL
-	resourceArns, err := service.ListResourcesForWebACL(ctx, &wafv2.ListResourcesForWebACLInput{WebACLArn: webACL.ARN}, func(options *wafv2.Options) {
-		options.Region = client.Region
-	})
-	if err != nil {
-		return diag.WrapError(err)
+	resourceArns := []string{}
+	if client.WAFScope == types.ScopeCloudfront {
```

Comment:
How/are we ensuring that when the scope is `CLOUDFRONT` the region is `us-east-1`?

Reply:
We do that in the PR from Timur https://github.com/cloudquery/cq-provider-aws/pull/675
The Multiplexer takes care of this to basically call `us-east-1` twice each time with a different scope.